puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -23,29 +27,18 @@
     notify  => Exec["jenkins-jobs-update-${config_name}"],
   }
 
-  cron { "jenkins-jobs-update-${config_name}-delete-old":
-    ensure      => absent,
-    command     => "timeout 1h jenkins-jobs --conf ${inifile} update --delete-old ${directory}/${config_name} > /var/cache/jjb.xml",
-    hour        => 0,
-    minute      => 10,
-    environment => 'PATH=/bin:/usr/bin:/usr/sbin',
-    require     => File[$inifile],
+  git::repo { "jenkins-jobs-${config_name}":
+    target => "${directory}/${git_project_name}",
+    source => $git_repo,
+    args   => $git_args,
   }
 
   exec { "jenkins-jobs-update-${config_name}":
     command => "jenkins-jobs --conf ${inifile} update ${directory}/${config_name} ${command_arguments}",
```

Comment:
this seem to contradict the ` cwd     => "${directory}/${git_project_name}/${git_relative_path}/${config_name}",` below? which will win?